examples/angular/package.json

```diff
@@ -13,44 +13,50 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "~11.2.0",
-    "@angular/core": "~11.2.0",
-    "@angular/forms": "~11.2.0",
-    "@angular/platform-browser": "~11.2.0",
-    "@angular/platform-browser-dynamic": "~11.2.0",
-    "@angular/router": "~11.2.0",
-    "@capacitor/core": "2.4.7",
-    "@capacitor/ios": "^2.4.7",
-    "@ionic/angular": "^5.5.2",
-    "@ionic/storage-angular": "file:../../angular/dist",
```

Comment:
I think we still want `file:../../angular/dist` otherwise it will resolve the latest published version vs. the local changes. 

This repo isn't using npm/yarn workspaces to resolve packages, so it is uses the `file:` resolution with lerna.

Reply:
file:../../angular/dist added back & chore added to subject line